code/modules/awaymissions/mission_code/ruins/oldstation.dm

```diff
@@ -43,7 +43,7 @@
 /obj/item/card/id/away/old/apc
 	name = "APC Access ID"
 	desc = "A special ID card that allows access to APC terminals."
-	icon_state = "centcom_old"
+	icon_state = "guest_invalid"
```

Comment:
To clarify for this one, `centcom_old` isn't an icon state anymore so it just showed the entire file when you examined it.